### PR TITLE
updates/testing: update to 30.20190905.0 release

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: updates/testing.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "stream": "testing",
   "metadata": {
-    "last-modified": "2019-09-09T10:01:35Z"
+    "last-modified": "2019-09-10T13:49:17+00:00"
```
Comment (dustymabe) [inline updates/testing.json]: the instructions say `date -u -Is` which gives me this format, which is not consistent with what we had in the past

Comment (jlebon) [reply updates/testing.json]: In cosa, we use `date -u +%Y-%m-%dT%H:%M:%SZ`.

**Comment by lucab** [comment]
LGTM.
Double-checking: a rollout for `30.20190905.0` will start on `Tue Sep 10 14:30:00 UTC 2019` and proceed from `0.0` to `1.0` in 24 hours (`1440` minutes)

```
$ date -u -d @1568125800
Tue Sep 10 14:30:00 UTC 2019

```

**Comment by dustymabe** [comment]
> LGTM.
> Double-checking: a rollout for `30.20190905.0` will start on `Tue Sep 10 14:30:00 UTC 2019` and proceed from `0.0` to `1.0` in 24 hours (`1440` minutes)

ack

**Comment by lucab** [comment]
Intermediate checkin: rollout correctly started on time. After a ~10 minutes, it's at 0.5% (`rollout_wariness=0.005`).

**Comment by lucab** [comment]
Intermediate checkin: we are experiencing a cluster-wide downtime (which is being investigated right now https://pagure.io/fedora-infrastructure/issue/8194).

The rollout is progressing (in theory) within the allotted time-window, but the service is down thus in practice nodes are not auto-updating. Additionally, 
For this reason (and more), in the future we plan to have an higher level human-friendly input (YAML, likely) which allows keeping track of pause/resume points.

**Comment by lucab** [comment]
Final checkin: `30.20190905.0` is now fully rolled out. I have an unattended canary node on AWS that correctly finalized the update during the night (UTC).

**Pull request merged**